KBaseFeatureValues.spec

```diff
@@ -111,6 +113,7 @@ module KBaseFeatureValues {
         ws_conditionset_id conditionset_ref;
         mapping<string, string> condition_mapping;
 
+        string        diff_expr_matrix_ref;
```

Comment:
Sorry I don't follow that at all

Reply:
https://ci.kbase.us/services/ws/docs/devtypedobjects.html#to-to-to-references-id-ws

The gist is that we might want to define a new type and use the `@id ws KBaseFeatureValues.DifferentialExpressionMatrix` as a decorator. This will tell the WS to ensure the passed string is a valid DEMatrix ref

Reply:
whats the syntax for that?

Reply:
ie do I just put that in a comment above the ExpressionMatrix struct declaration?

Reply:
Rather then define diff_expr_matrix_ref as type string make a new type for diff_expression_matrix_refs like was done for this type for genome_refs https://github.com/kbaseapps/FeatureValues/blob/master/KBaseFeatureValues.spec#L19-L23